frontend/src/pages/Admin/PickerOptions/ProjectBillingResourceTypeOption/tableConfig.ts

```diff
@@ -0,0 +1,8 @@
+import { GridColDef } from "@mui/x-data-grid";
+
+export const tableConfig = () => {
+  const defaultFlex = 3;
+  const tableColumns: GridColDef[] = [{ field: "label", headerName: "Label", flex: defaultFlex }];
+
```

Comment:
is the tableConfig the same for all of these, I wonder?

Reply:
it is, again will change in the future so I'm setting up future devs.